meetings/2017-08-10.md

```diff
@@ -0,0 +1,98 @@
+# Security WG meeting 2017-08-10
+
+GitHub issue: https://github.com/nodejs/security-wg/issues/32
+Meeting video: TBA
+Previous meeting: https://github.com/nodejs/security-wg/pull/28
+
+
+# Present
+
+- Josh Brown-White (@joshbw)
+- Colin Ihrig (@cjihrig)
+- Reed Loden (@reedloden)
+- Deian Stefan (@deian)
+- Michiel Prins (@michiel3)
+- Bryan English (@bengl)
+- Devon Rifkin (@drifkin)
+- Sam Roberts (@sam-github)
+
+# Review of last meeting
+
+Actions from https://github.com/nodejs/security-wg/blob/ac6623f305002b973011d58f4e2485a0f1e2a0b5/meetings/2017-07-13.md#actions
+
+[ ] Michael: will document the security release process
+[ ] Michael: will add recurring sec-wg meeting to calendar
+[x] Sam: will schedule the next WG meeting (https://github.com/nodejs/security-wg/issues/32)
+[x] Sam: will look at https://github.com/rubysec/ruby-advisory-db to see how they store vulnerabilities for gems and ruby (see end of file)
+[x] Sam: will PR a description of what Node.js considers a security issue (https://github.com/nodejs/security-wg/issues/18) See https://github.com/nodejs/node/pull/14485
+[x] JoshBW: will evaluate HackerOne and BugCrowd (https://github.com/nodejs/security-wg/issues/16, https://github.com/nodejs/security-wg/issues/17)
+[x] JoshBW: will investigate what it takes to get a semmle.com license for Node.js as an open source project (https://github.com/nodejs/security-wg/issues/29)
+
+# Agenda
+
+## HackerOne Demo
+
+Reed gave a demo of HackerOne’s dashboard.  He sent out a bunch of demo
+accounts.  If you didn’t get an email and would like a demo account please ping
+@reedloden (reed @ hackerone.com)
+
+Can’t set an “auto-disclosure” after X days of filings in the settings (though
+you can after a bug was resolved), but probably possible with the API.
+Normally disclosure requires an intentional act within the report UI.
+
+Full API (https://api.hackerone.com).  Probably not a Node.js library for it
+yet (Ruby, Python, Go, etc. are present)
+
+Can issue CVEs for both node and npmjs.org packages if we choose to go down
+that route.
+
+Two possible use-cases:
+1. Node vulnerabilities
+2. Npmjs.org package vulnerabilities
+
+For npmjs.org packages, HackerOne looks to have a great feature set, CVE
+issuance and the rich permission model among them.
+
+
+I’d like to see some investigation of the API to ensure we can pull issues out
+as JSON for storage in github.com, but otherwise I think we should try it out.
```

Comment:
I know that GitHub's security team uses our API to pull content in order to display on https://bounty.github.com/, so it's definitely possible. :)